test/k8sT/istio.go

```diff
@@ -82,11 +82,18 @@ var _ = Describe("K8sIstioTest", func() {
 
 		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
 
-		err := kubectl.CiliumInstall(helpers.CiliumDefaultDSPatch, helpers.CiliumConfigMapPatch)
+		_ = kubectl.Apply(helpers.DNSDeployment())
+
+		// Deploy the etcd operator
```

Comment:
This comment does not add a real value, the function is DeployETCDoperator, is self descriptive.